server/src/main/java/com/tencent/rss/server/ShuffleServerGrpcService.java

```diff
@@ -345,18 +348,27 @@ public void getShuffleResult(GetShuffleResultRequest request,
       LOG.error("Error happened when get shuffle result for " + requestInfo, e);
     }
 
-    if (serializedBlockIds == null) {
-      serializedBlockIds = new byte[]{};
+    GetShuffleResultResponse.Builder replyBuilder = GetShuffleResultResponse.newBuilder().setRetMsg(msg);
```

Comment:
why this method should be updated?

Reply:
getShuffleResult and getShuffleData are both update for the copyFrom is replace by unsafeWrap to avoid bytes copy.